### PR TITLE
P0 bug fixes

### PR DESCRIPTION
**Pull request by innopoop**
Show admins in settings page
Remove double click to access program homepage from user homepage
Correct functionality of /apply page back button

### CODE REVIEW

**Comment by nova-admin** [comment]
#### 📝Changed routes:


| [**/create**](https://risc-ufo5vs050-novaforgood.vercel.app/create) | [**/**](https://risc-ufo5vs050-novaforgood.vercel.app/) |
|:-:|:-:|
|<a href="https://risc-ufo5vs050-novaforgood.vercel.app/create">Screenshot of /create</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/zDfSvVmO-twffLVN50RW3VIHqRFfKLEXLleDJ8gBDaEMhAahnieVRFhuzU-A0JSQRdpK779SLm9qRYlsRLiHBg.png">(view full size)</a> |<a href="https://risc-ufo5vs050-novaforgood.vercel.app/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/rkPD8jhi01ALgXkPVLsq_yLizIUNxUAE__UVIluH1bj3WZi-QItFQ1efNQydba7w7Hx23rp6BxIwQ85TOrHX_Q.png">(view full size)</a> |



| [**/login**](https://risc-ufo5vs050-novaforgood.vercel.app/login) | [**/program/\[slug\]/apply**](https://risc-ufo5vs050-novaforgood.vercel.app/program/[slug]/apply) |
|:-:|:-:|
|<a href="https://risc-ufo5vs050-novaforgood.vercel.app/login">Screenshot of /login</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/UfpdZ2AELoYcqvOAi8Fh-szgIJxyRybu22k9mAxj37XqhDMzRtqwcndwoH1JANtWTYEraj3MkNx_RAL0aPAltA.png">(view full size)</a> |<a href="https://risc-ufo5vs050-novaforgood.vercel.app/program/[slug]/apply">Screenshot of /program/[slug]/apply</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/yUBSwZakqDdAtOiIJ44tEiZm_npYj6fkzELyoRl-FuTj64Y9GWU-iaMedcaaip1X-9uyxXoWlCCREIrfaqADGw.png">(view full size)</a> |



| [**/program/\[slug\]/settings**](https://risc-ufo5vs050-novaforgood.vercel.app/program/[slug]/settings) | [**/signup**](https://risc-ufo5vs050-novaforgood.vercel.app/signup) |
|:-:|:-:|
|<a href="https://risc-ufo5vs050-novaforgood.vercel.app/program/[slug]/settings">Screenshot of /program/[slug]/settings</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/HcNiQSxX7Kywya6yIVplB6jLjakYJKp57CnmAzWR9OycOgSVANtmPGCZdW6ENPbEPy7Sq_5zNvUZJI-KuRv3Gg.png">(view full size)</a> |<a href="https://risc-ufo5vs050-novaforgood.vercel.app/signup">Screenshot of /signup</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/vuL0GtJugKuvBdxu8qSOfwH8ChepaMlPBg8YKuYg7G6iZM0f3H1hS9ztLGIZ7jwk9hkZ5RqE9bSXvdDtgrfO9Q.png">(view full size)</a> |



And 1 other route:
- [**/verify**](https://risc-ufo5vs050-novaforgood.vercel.app/verify)



Commit 2377c6487ec161561f34f731b3fa791752c5d2ea (https://risc-ufo5vs050-novaforgood.vercel.app).

**Review by legitmaxwu — APPROVED** [review]
LGTM. For copying links, is it possible to auto-highlight the selected text when you click copy? Otherwise, there's no clear indication that the copy went through.

File: src/components/ProfileModal.tsx [hunk 1]
```diff
@@ -172,7 +164,6 @@ const ProfileModal = ({
             >
               Back
             </Button>
-            <BookAChat mentor={profile} />
```
Comment (legitmaxwu) [inline src/components/ProfileModal.tsx:175]: Why is this removed?

**Review by legitmaxwu — CHANGES_REQUESTED** [review]

File: src/pages/signup.tsx [hunk 2]
```diff
@@ -218,7 +204,10 @@ const SignUpPage: Page = () => {
                         }),
                       ])
                         .then(() => {
-                          router.push("/verify");
+                          router.push(
```
Comment (legitmaxwu) [inline src/pages/signup.tsx:207]: I think the following is possible with the Next.JS router.

```
router.push({
    pathname: '/verify',
    query: { to: nextURL }
})
```

**Review by legitmaxwu — CHANGES_REQUESTED** [review]

File: src/components/ProfileModal.tsx [hunk 3]
```diff
@@ -48,14 +48,14 @@ const ProfileModal = ({
...
   const { currentProgram } = useCurrentProgram();
-
   const authorizationLevel = useAuthorizationLevel();
 
   useEffect(() => {
     if (isOpen === true) setStage(ProfileModalStage.VIEW_PROFILE);
     return () => {};
   }, [isOpen]);
 
+  console.log(profile);
```
Comment (legitmaxwu) [inline src/components/ProfileModal.tsx:58]: Remove

File: src/components/ProfileModal.tsx [hunk 4]
```diff
@@ -90,26 +90,28 @@ const ProfileModal = ({
...
-                    Book a Chat
-                  </Button>
-                  {/* <div className="w-2"></div>
+                {/* <div className="flex "> */}
+                <Button
+                  size="small"
+                  onClick={
+                    authorizationLevel === AuthorizationLevel.Mentee
+                      ? () => {
+                          setStage(ProfileModalStage.BOOK_CHAT);
```
Comment (legitmaxwu) [inline src/components/ProfileModal.tsx:99]: nit: Ternary might be a little confusing here. I'd just do 

```
if (authorizationLevel === AuthorizationLevel.Mentee) // book chat
```

File: src/components/ProfileModal.tsx [hunk 5]
```diff
@@ -90,26 +90,28 @@ const ProfileModal = ({
                 </div>
                 <div className="h-4"></div>
 
-                <div className="flex ">
-                  <Button
-                    size="small"
-                    disabled={authorizationLevel !== AuthorizationLevel.Mentee}
```
Comment (legitmaxwu) [inline src/components/ProfileModal.tsx:96]: The button should be `disabled` for Admins

**Review by legitmaxwu — APPROVED** [review]
Looks good, nice work! Just a few more notes but then it should be good to merge!

**Pull request merged**